blockware/lib/VirtualBlox/stubs/Adafruit_GFX.cpp

```diff
@@ -34,7 +34,6 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "Adafruit_GFX.h"
 #include "glcdfont.c"
 #include "utf8_decode.h"
-#include "utf8_decode.c"
```

Comment:
I think the reason @magus you had to do this is that the symbols in C files (`.c`) don't automatically get shared with the symbols in C++ files `.cpp`. There's a way around this (something about `extern "C"` I think) or we can just name all our files `.cpp`. Does this sound right to you @mikeperiscope

Reply:
Sounds right, if you try to include things built by the c compiler (.c files, generally) they have different symbol names than if you build with a c++ compiler. You need to either include them in an extern "C" block or build them with a c++ compiler to have the functions get found